packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -543,6 +543,7 @@ export const SubmitButtonWrapper = styled.div`
 
 	.checkout__step-wrapper--last-step & {
 		position: fixed;
+		box-shadow: 0 -3px 10px 0 #0000001f;
```

Comment:
I think the `rgba` notation is preferred `rgba( 0, 0, 0, 0.12 )`. I don't know if that's documented, but I can't find any patterns matching `#([A-Fa-f0-9]{8})$` in Claypso.

Reply:
FWIW I think that including alpha in a hex color is a well-supported standard: https://caniuse.com/?search=alpha%20hexadecimal

Reply:
I pulled this value from the class used in onboarding, so I was just trying to preserve the style they used.